TeamMemberFiles/helen.mills.txt

```diff
@@ -0,0 +1,5 @@
+Helen Mills
+Please call me Helen (she/her)
+I'm in section 294
+I don't really know many languages well enough to prefer them, but I 
+am most familiar with SQL, Java, HTML.
```

Comment:
Awesome.  Please make sure to clean this up on your repo so that you don't expose any private information to the public.